chipsec/chipset.py

```diff
@@ -243,6 +243,12 @@ def init( self, platform_code, req_pch_code, start_driver, driver_exists=None, t
                 self.vid = vid
                 self.did = did
                 self.rid = rid
+            elif cpuid in self.detection_dictionary.keys():
+                self.code       = self.detection_dictionary[cpuid]
```

Comment:
Do we need `_unknown_platform = False` ?

Reply:
Thanks :)